src/_base/harness/config/pipeline.yml

```diff
@@ -91,6 +91,9 @@ command('app deploy <environment>'):
     if helm version --short --client | grep '^Client: v2' >/dev/null 2>&1; then
       passthru helm init --client-only
     fi
+    if helm version --short --client | grep '^v3' >/dev/null 2>&1; then
+      kubectl --kubeconfig="${PWD}/kubectl.config.yaml" create ns "${NAMESPACE}" || true
+    fi
```

Comment:
needs to be conditional for if the k8s server doesn't allow ns creation to the user. Not all uses will be necessarily for feature branch environments only